### PR TITLE
Run just native tests in CI Native builds

### PR DESCRIPTION
**Pull request by rsvoboda**
Run just native tests in CI Native builds

(plus make sure NativeDebugTest runs just on Linux)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
             name: ci-artifacts
             path: artifacts-native${{ matrix.java }}.zip
   build-code-start:
```
Comment (jsmrcka) [inline .github/workflows/ci.yaml:91]:
```suggestion
  linux-build-code-start:
```
Just to make it consistent with the others...

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -89,7 +89,7 @@ jobs:
...
-    name: Code Quarkus build - released Quarkus
+    name: Code Quarkus build
```
Comment (jsmrcka) [inline .github/workflows/ci.yaml:92]:
```suggestion
    name: Linux - Code Quarkus build
```
Again, consistency, but I'm not sure about this one, as it makes the name longer.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -27,7 +27,7 @@ jobs:
         run: |
           mvn -V -B -s .github/mvn-settings.xml validate -Pide,validate-format
   linux-build-released-jvm:
```
Comment (jsmrcka) [inline .github/workflows/ci.yaml:29]: Do we want to keep the `released` infix here (and in all the other jobs)?

**Review by jsmrcka — COMMENTED** [review]
Found just a few details, otherwise looks good.

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (rsvoboda) [reply .github/workflows/ci.yaml:29]: imho not needed, we need just the names shorter to see the details in checks section bellow

**Comment by rsvoboda** [comment]
@jsmrcka ready for second round

**Review by jsmrcka — APPROVED** [review]

**Pull request merged**